scripts/requirements-test.yml

```diff
@@ -11,7 +11,7 @@
 # limitations under the License.
 # Authors:
 # CIL Developers, listed at: https://github.com/TomographicImaging/CIL/blob/master/NOTICE.txt
-name: cil_dev
+name: cil_dev2
```

Comment:
I guess you didn't mean to commit this change.

Reply:
Oops, yes you're right - thank you for catching that!